Car_Collision_Visulaization/main.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include "src/data_visualization.h"
+#include "src/car_collision.h"
+
+
+
+int main() 
+{
+
+    Car car1(200, 2, 5, 45 );    //angle in radian    
+    Car car2(55 , 4, 7, 135);  
+    Visualizer visualize; 
+ 
+    // Car car1(2, 2, 5, 45 );    //angle in radian    
+    // Car car2(5, 4, 7, 135); 
+    // Visualizer visualize;
+
+    visualize.update(car1,car2); 
```

Comment:
Initialize all cars in a vector .

std::vector<Car> all_cars;
all_cars.push_back(200,2,5,45);
all_cars.push_back(55,4,7,135);

while (true)
{ 
  for (const Car& car : all_cars)
    car.move();
   visualize.update(all_cars)
}

Reply:
visualize object needs default constructor there